src/main/java/com/thebois/models/world/World.java

```diff
@@ -50,4 +55,25 @@ public Object find() {
         return copy;
     }
 
+    /**
+     * Returns the players' colony.
+     *
+     * @return the colony.
+     */
+    public IBeingGroup getColony() {
+        return this.colony;
+    }
+
+    /**
+     * Updates the state of the world.
+     */
+    public void update() {
```

Comment:
Feels a bit wrong to have a colony update here but I understand why you placed it here.